docs/API.md

```diff
@@ -366,3 +367,8 @@ An array of regexp pattern strings that are matched against all modules before t
 This is useful for some commonly used 'utility' modules that are almost always used as implementation details almost all the time (like underscore/lo-dash, etc). It's generally a best practice to keep this list as small as possible and always use explicit `jest.mock()`/`jest.dontMock()` calls in individual tests. Explicit per-test setup is far easier for other readers of the test to reason about the environment the test will run in.
 
 It is possible to override this setting in individual tests by explicitly calling `jest.mock()` at the top of the test file.
+
+### `config.verbose` [boolean]
+(default: `false`)
+
+Indicates whether the tests should be printed into the console using `Mocha`-styled logs for determining if a specific test has failed or succeeded. Errors will still show on the bottom after execution.
```

Comment:
In general, it's not specifically "Mocha-styled" (though we took the "UI" from there). I think we can omit this part.